src/Workspaces/Core/Portable/Storage/SQLite/Interop/SafeSqliteStatementHandle.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+using SQLitePCL;
+
+namespace Microsoft.CodeAnalysis.SQLite.Interop
+{
+    internal sealed class SafeSqliteStatementHandle : SafeSqliteChildHandle<sqlite3_stmt>
+    {
+        public SafeSqliteStatementHandle(SafeSqliteHandle sqliteHandle, sqlite3_stmt? wrapper)
+            : base(sqliteHandle, wrapper?.ptr ?? IntPtr.Zero, wrapper)
```

Comment:
should the `?? IntPtr.Zero` just happen in the base class instead of all the derived ones?

Reply:
The `ptr` property isn't defined in an interface, and it seemed weird(er) to pass down `IntPtr?`.